jupyter_server/transutils.py

```diff
@@ -10,4 +10,4 @@
 # Set up message catalog access
 base_dir = os.path.realpath(os.path.join(__file__, '..', '..'))
 trans = gettext.translation('notebook', localedir=os.path.join(base_dir, 'notebook/i18n'), fallback=True)
-_ = trans.gettext
+_i18n = trans.gettext
```

Comment:
@goanpeca, @zsailer - do you know if server extensions use _this_ `_` alias to wrap localizable strings?  If so, we may not want to fully replace it, but define two aliases for now.  Since we may not know, it might be best to keep both.

@sngyo - I suspect we'll want to add some kind of commented directive such that the "unused global variable" warning is not emitted.  (This is probably another reason why `_` was used.)

Reply:
@kevin-bates 
### 1. two alias
How do you think introducing a deprecation warning for `_()` as below?
```python3
import warnings

def _trans_gettext_helper(*args, **kwargs):
    warn_msg = "The alias `_()` will be deprecated. Use `_i18n()` instead."
    warnings.warn(warn_msg, UserWarning)
    return trans.gettext(*args, **kwargs)


_ = _trans_gettext_helper
_i18n = trans.gettext
```

### 2. skip `unused global variable`
I'll look for the method to ignore (skip) the warning for Github CodeScanning. There should be something like this (maybe it will work 😆 ):
```python
_i18n = trans.gettext  # noqa: E640
```

Reply:
> 1. two alias

I think it makes sense to add a deprecation warning for this.  I see you're using `UserWarning` - I'm guessing due to the limitations for `DeprecationWarning`.  I would suggest using `FutureWarning` for this instead since it more closely identifies with the intention.  (We also used this warning level recently in jupyter_client.)

> 2. skip unused global variable

I had _linting_ on my mind when I made this suggestion.  I'm not immediately finding linting-style directives for codeQL, so this probably isn't a big deal since codeQL is still succeeding.  Sorry for the misdirection.